gp-settings.php

```diff
@@ -180,3 +177,6 @@ function gp_run_route() {
 	}
 }
 add_action( 'template_redirect', 'gp_run_route' );
+
+// Load the plugin's translated strings
+load_plugin_textdomain( 'glotpress', false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
```

Comment:
Should we use `GP_LANG_PATH` here? If not, then we should remove the constant, line 8 in this file.